templates/firefox-ios.xliff

```diff
@@ -3523,6 +3543,10 @@ Private Tab</source>
         <source>Make %@ your go-to browser</source>
         <note>String used to describes the title of what Firefox is on the welcome onboarding page for current version in our Onboarding screens. Placeholder is for the app name.</note>
       </trans-unit>
+      <trans-unit id="Onboarding.Welcome.Title.TreatementA.v120" xml:space="preserve">
+        <source>We love keeping you safe</source>
+        <note>String used to describes the title of what Firefox is on the welcome onboarding page for current version in our Onboarding screens. Placeholder is for the app name.</note>
```

Comment:
Nit for comment: there's no placeholder in the string

Reply:
resolved